robottelo/cli/metatest/__init__.py

```diff
@@ -3,9 +3,8 @@
 import itertools
 import types
 
-from ddt import ddt
+from ddt import ddt, data
```

Comment:
I dropped our data decorator, and is using ddt data. I have incorporated changes to drop ddt so far. But I don't looked if data is really necessary or not on the modules since this is not the goal of this PR.